packages/react/src/components/InlineAlert/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { Icon } from "..";
+import { INLINE_ALERT_TYPES, ICONS } from "../../constants/";
+
+const ALERT_ICONS = {
+  [INLINE_ALERT_TYPES.INFORMATION]: ICONS.IconInfo,
+  [INLINE_ALERT_TYPES.SUCCESS]: ICONS.IconCheck,
+  [INLINE_ALERT_TYPES.ERROR]: ICONS.IconStop,
+};
+
+const ALERT_TYPES = Object.values(INLINE_ALERT_TYPES);
+
+const InlineAlert = ({ title, message, type, onClose, ...props }) => (
+  <div className={cx("InlineAlert", `InlineAlert--${type}`)} {...props}>
+    {onClose && (
+      <span className="InlineAlert-iconClose" onClick={onClose}>
+        <Icon icon={ICONS.IconClose} />
+      </span>
+    )}
+    <div className="InlineAlert-icon">
+      <Icon icon={ALERT_ICONS[type]} />
+    </div>
+    <div>
+      <div className="InlineAlert-title">{title}</div>
```

Comment:
But styles of `h1` are completely different than these that are expected by design :(

![image](https://user-images.githubusercontent.com/15340486/60808955-c806ef80-a189-11e9-9a7e-19eb43b52701.png)

![image](https://user-images.githubusercontent.com/15340486/60808981-d6eda200-a189-11e9-92c5-df7488885df1.png)

Reply:
CSS is your friend 😁

Reply:
So that's what I'm trying to understand. Of course it's possible to style it, but what's the value of changing it to `h1`?

What I'm thinking is that `div` works just as fine, and when I change it to `h1` there's an additional work I need to do. And this will repeat for all upcoming occurrences :/

Reply:
What I'm sensing is that you might want to have it this way here to have it done `by the book` and as everywhere else, so that we maintain this library mind-blowing. Could I be right? :)

If that's the case it's ❤️ but maybe in that particular case it's not worth the effort, hmmm? :P

Reply:
It's worth the effort because it's for an accessibility matter! Otherwise, let's use `div`s and `span`s everywhere and we don't care 🤷‍♂ 

Accessibility is important.
We want our products to be accessible by as many people possible, whatever their condition (under a reasonable bar). And it all starts with this first-level components library.
If it means spending 5 additional minutes to add some CSS to match the designs, yeah we should do it.

Reply:
Accessibility is important - no doubt. How does `<p>` help in that?

Reply:
Done!

Reply:
Did you forget to push your commit? 😛